.travis.yml

```
@@ -50,3 +50,4 @@ after_failure:
   - pip --version
   - pip list
   - docker logs itest-yarn
+  - cat /tmp/kernel*.log || true
```

Since the logs are in the container, I think this might need to be something like:
```suggestion
  - docker exec -it itest-yarn cat /tmp/kernel*.log || true
```